buildSrc/src/main/groovy/Venv.groovy

```diff
@@ -570,39 +570,6 @@ This task will dependsOn other tasks which actually perform testing.'''
             // clean out the reporting directory before we run.
             project.delete(reportDir)
         }
-        handleFailure(project, e, new File(reportDir))
         return e
     }
-
-    //TODO core#430 delete when this task is closed out
-    void handleFailure(Project project, Exec e, File reportDir) {
-        Gradle gradle = project.gradle
-        boolean allowFailure = TestTools.allowFailure(gradle.rootProject)
-        if (gradle.startParameter.continueOnFailure) {
```

Comment:
You are going to still want to support continueOnFailure for python tests; they will completely break rather than just "let the junit xml turn into a failed test / warning / yellow result".

Reply:
I'm confused - by ripping this code out, I was assuming that my new logic (none) would do the same thing as the existing code, since the `continueOnFailure` block just returns...

Reply:
Yes, I think you're right @devinrsmith - as long as `--continue` is passed, the log-or-throw will be skipped entirely? This _is_ the handler for the python tests. As the next line says the exec will throw, and so `--continue` will keep running more tasks that don't depend on this one, the doList { buildFinished {...}} lambdas only kick in if `--continue` _isnt_ passed (in which case we expect the build to fail right away, which is the default in lieu of an "allowFailure" option?